### DISCUSSION TITLE
Works with Capacitor

### DISCUSSION THREAD

**Discussion by taylordwright44** [pinecone-router/router#35] [Show and tell]
I made an app with Alpine+Bulma+Pinecone on the front end and Mongo+Express+Node on the backend. I decided to see what would happen if I copy pasted the entire front end into Capacitor and after a bit of tweaking the fetch URLS to point at my backend server I got a fast and clean Android app out of it. Thought I'd show it off in case anyone else ever thought of making a phone app so they'd know it works. Capacitor is made by the Ionic team btw for anyone who doesn't know, Ionic is just a wrapper so you can use Angular/React/Vue on Capacitor. So that makes Capacitor perfect for Alpine+Pinecone. Bulma is nice too because it makes the app resize nicely with built in flexbox. This is my new favorite stack, 10/10 project.

[cinnamon-20240326-1.webm](https://github.com/pinecone-router/router/assets/11287407/0b776107-d6ad-4274-b903-9d2e46760650)

**Comment by rehhouari** [comment]
That's great to hear thanks alot for sharing! it does look snappy indeed glad it worked. Feel free to open any issues for problems that come your way.
I'll give this a try aswell when I I want to make a mobile app.

**Comment by bsp982** [comment]
excellent development
